src/sparql/get-terms-by-regex.sparql.jinja2

```diff
@@ -0,0 +1,9 @@
+prefix owl:  <http://www.w3.org/2002/07/owl#>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+
+select ?term_id ?term_label where {
+  ?term_id a owl:Class;
+    skos:prefLabel ?term_label .
+  FILTER (REGEX(?term_label, "{{ regexp }}"))
+  # FILTER(isIRI(?term_id) && (STRSTARTS(str(?term_id), "{{ iri }}")))  # leaving this out and doing in python for now
```

Comment:
I created a new SPARQL Jinja template for REGEX queries.

In a different review comment, I mentioned that there could be a problematic edge case where sometimes terms that are xrefs from other ontologies might be caught and returned by the regexp pattern. I'm not doing this currently, though.